content/guides/references/error-messages.md

```diff
@@ -455,23 +455,50 @@ it('does not forget to return a promise', () => {
 })
 ```
 
-### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` failed because you are attempting to visit a second unique domain
+### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` succeeded, but commands are timing out because you visited a different origin domain without using `cy.origin()`
 
-::include{file=partials/single-domain-workaround.md}
+As of Cypress [v12.0.0](https://on.cypress.io/changelog#12-0-0), users can
+navigate to multiple domains in a single test. The following test will succeed
+as is:
 
-### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` failed because you are attempting to visit a different origin domain
+```javascript
+it('navigates to docs.cypress.io', () => {
+  cy.visit('http://localhost:3000') // where your web server + HTML is hosted
+  cy.visit('https://docs.cypress.io') // visiting a different superdomain
+})
+```
+
+However, when the newly visited URL is not considered the same superdomain, the
+[`cy.origin()`](/api/commands/origin) command **must** be used to interact with
+the newly visited domain. The following test is incorrect:
+
+```javascript
+it('navigates to docs.cypress.io and runs additional commands', () => {
+  cy.visit('http://localhost:3000')
+  cy.visit('https://docs.cypress.io') // visiting a different superdomain
+  cy.get('h1').should('contain', 'Why Cypress?') // run new assertions fail
+})
+```
+
+And will result in the following error:
 
-::include{file=partials/single-domain-workaround.md}
+<DocsImage src="/img/guides/references/cy-visit-subsequent-commands-timed-out.png" alt="cy.visit() subsequent commands timed out" ></DocsImage>
```

Comment:
Won't this be out of date once we update to have a doc link to origin?

Reply:
I don't think so, since it explains why the error happens, which we have done with the visiting behavior in the past

Reply:
ohh I thought that's what Jennifer was pointing out earlier 😅

Reply:
Oh I think you mean the image? yeah that's gonna have to get updated. My bad 😆

Reply:
oops...yeah! sorry. I couldn't comment on the image. I should have been clearer

Reply:
image updated in 5ed169d5f